cmd/sansshell-server/default-policy.rego

```diff
@@ -63,3 +63,8 @@ allow {
 allow {
 	input.type = "Service.StatusRequest"
 }
+
+denial_hints[msg] {
+	#input.message.file.filename != "/etc/hosts"
+	msg := "we only allow /etc/hosts"
```

Comment:
If I understand this correctly, the denial hint should be placed right after the corresponding rule. So, shouldn't this be at L32?

Reply:
One of the weird things about this language is that rule order tends to not matter. Placing after the other rule or at the end both work fine.

https://www.openpolicyagent.org/docs/latest/faq/#statement-order